mermithid/processors/IO/MultiChannelCicadaReader.py

```diff
@@ -0,0 +1,108 @@
+'''
+Processor that can read (not write) Katydid ROOT files using the Cicada library
+Author: C. Claessens
+Date: April 08 2020
+'''
+
+from morpho.processors.IO import IOProcessor
+from morpho.utilities import reader, morphologging
+logger = morphologging.getLogger(__name__)
+
+try:
+    from ROOT import TFile, TTreeReader, TTreeReaderValue
+except ImportError:
+    pass
+
+import numpy as np
+
+class MultiChannelCicadaReader(IOProcessor):
+    '''
+    Processor that can read (not write) Katydid ROOT files using the Cicada library
+    '''
+
+    def InternalConfigure(self,params):
+        '''
+        Args:
+            object_type: class of the object to read in the file
+            object_name: name of the tree followed by the name of the object
+            use_katydid: retro-compatibility to Katydid namespace
+        '''
+        super().InternalConfigure(params)
+        self.object_type = reader.read_param(params,"object_type","TMultiTrackEventData")
+        self.object_name = reader.read_param(params,"object_name","multiTrackEvents:Event")
+        self.use_katydid = reader.read_param(params,"use_katydid",False)
+
+        self.N_channels = reader.read_param(params, "N_channels", 3)
+        self.channel_ids = reader.read_param(params, "channel_ids", ['a', 'b', 'c'])
+        self.rf_roi_min_freqs = reader.read_param(params, "rf_roi_min_freqs", [0, 0, 0])
+        self.transition_freqs = reader.read_param(params, "channel_transition_freqs", [0, 0])
+        self.frequency_variable_name = reader.read_param(params, "merged_frequency_variable", "F")
+        return True
```

Comment:
Maybe add a check that the number of files corresponds to the number of channels to read?